desktop/js/scenario.js

```diff
@@ -2185,7 +2185,11 @@ document.getElementById('scenariotab').addEventListener('click', function(event)
     return
   }
 
-  if (_target = event.target.closest('#bt_cancelElementSave')) {
+  if (_target = event.target.closest('#bt_cancelElementSave'))  {
+    jeeDialog.modal(document.getElementById('md_addElement'))._jeeDialog.hide()
+  }
+
+  if (_target = event.target.closest('#bt_crossElementSave'))  {
```

Comment:
It's good practice to use strict equality (===) instead of loose equality (==) in JavaScript to avoid unexpected type coercion. [important]

Reply:
Line 2192 is an affectation combined with a null/undefined check... 
No need for `===` or `==` here.